docs/source/user_guide/plots.rst

```diff
@@ -0,0 +1,969 @@
+.. _interactive-plots:
+
+Interactive Plots
+=================
+
+.. default-role:: code
+
+FiftyOne provides a powerful
+:mod:`fiftyone.core.plots <fiftyone.core.plots.base>` framework that contains
+a variety of interactive plotting methods that enable you to visualize your
+datasets and uncover patterns that are not apparent from inspecting either the
+:ref:`raw media files <fiftyone-app>` or
+:ref:`aggregate statistics <using-aggregations>`.
+
+With FiftyOne, you can visualize geolocated data on maps, generate interactive
+evaluation reports such as confusion matrices and PR curves, create dashboards
+of custom statistics, and even generate low-dimensional representations of your
+data that you can use to identify data clusters corresponding to model failure
+modes, annotation gaps, and more.
+
+What do we mean by **interactive plots**? First, FiftyOne plots are
+`powered by Plotly <https://plotly.com/python>`_, which means they are
+responsive JavaScript-based plots that can be zommed, panned, and lasso-ed.
+Second, FiftyOne plots can be linked to the :ref:`FiftyOne App <fiftyone-app>`,
+so that selecting points in a plot will automatically load the corresponding
+samples/labels in the App (and vice versa) for you to visualize! Linking plots
+to their source media is a powerful paradigm that it is a critical part of any
```

Comment:
Used powerful a lot, recommend changing this to:
"Linking plots to their source media is a compelling paradigm that is a critical part..."